Projects/UOContent/Items/Containers/TreasureMapChest.cs

```diff
@@ -361,7 +361,10 @@ private bool CheckLoot(Mobile m, bool criminalAction)
 
     public override void OnItemLifted(Mobile from, Item item)
     {
-        var notYetLifted = !_lifted.Contains(item);
+        
+        _lifted ??= new HashSet<Item>();
+        
+        var notYetLifted = !_lifted.Contains(item); 
```

Comment:
Let's add a null check here, and move the assignment to where we need it (above the add):

```cs
var notYetLifted = _lifted?.Contains(item) != true;
```